src/views/GeneralChargeConfig.vue

```diff
@@ -312,6 +379,8 @@ export default {
 	data() {
 		return {
 			mqttTopicsToSubscribe: [
+				"openWB/bat/config/power_limit_mode",
+				"openWB/bat/set/power_limit_controlable",
```

Comment:
Müsste das Topic nicht `../get/...` heißen? Das ist ja eine Eigenschaft, die das Speicher-Modul liefert. Keine Option, welche im Speicher gesetzt werden kann.
```suggestion
				"openWB/bat/get/power_limit_controlable",
```

Reply:
Stimmt, ist angepasst.